azure-pipelines.yml

```diff
@@ -19,10 +19,18 @@ container:
   image: golang:1.12.9-buster
 
 steps:
-- checkout: self
-  clean: true
-  fetchDepth: 1
+  - checkout: self
+    clean: true
+    fetchDepth: 1
 
-- script: |
-    set -eu -o pipefail
-    go test -race ./...
+  - script: go get -u golang.org/x/lint/golint
```

Comment:
Eventually we will want to do something with this. Pulling and building from master is not ideal.